backend/Squirrel.ConsoleApp.BL/Services/Abstract/BaseDbService.cs

```diff
@@ -31,13 +31,15 @@ private protected async Task<QueryResultTable> ExecuteQueryFromConnectionInterna
     {
         using var command = CreateCommandInternal(connection, query);
 
+        command.CommandTimeout = 30;
+
         await connection.OpenAsync();
         await using var reader = await command.ExecuteReaderAsync();
 
         var result = BuildTable(reader);
 
         return result;
-}
```

Comment:
by the way, we Open a connection here, but we don't close it. Maybe we should close it?

Reply:
Very good point, actually. Thanks!